src/check_ast.rs

```diff
@@ -451,6 +454,9 @@ where
                 if self.settings.enabled.contains(&CheckCode::B018) {
                     flake8_bugbear::plugins::useless_expression(self, body);
                 }
+                if self.settings.enabled.contains(&CheckCode::B021) {
```

Comment:
I think we also need to add this to `fn visit_docstring`, which checks the module-level docstring.